status-payloads-spec.md

```diff
@@ -114,15 +168,19 @@ Message types are required to decide how a particular message is encrypted and w
 
 
 The following messages types MUST be supported:
-* `public-group-user-message` is a message to the public group
-* `user-message` is a private message
-* `group-user-message` is a message to the private group.
+* `ONE_TO_ONE` is a message to the public group
+* `PUBLIC_GROUP` is a private message
+* `PRIVATE_GROUP` is a message to the private group.
 
 #### Clock vs Timestamp and message ordering
 
 `timestamp` MUST be Unix time calculated when the message is created in milliseconds. This field SHOULD not be relied upon for message ordering.
 
-`clock` SHOULD be calculated using the algorithm of [Lamport timestamps](https://en.wikipedia.org/wiki/Lamport_timestamps). When there are messages available in a chat, `clock`'s value is calculated based on the last received message in a particular chat: `last-message-clock-value + 1`. If there are no messages, `clock` is initialized with `timestamp * 100`'s value.
+`clock` SHOULD be calculated using the algorithm of [Lamport timestamps](https://en.wikipedia.org/wiki/Lamport_timestamps). When there are messages available in a chat, `clock`'s value is calculated based on the last received message in a particular chat: `max(timeNowInMs, last-message-clock-value + 1)`. If there are no messages, `clock` is initialized with `timestamp * 100`'s value.
```

Comment:
Why `* 100`?

Reply:
that's actually not the case anymore, we multiplied by `100` to separate the timestamp from the  `sequence` number of the message, but it's not really needed, so we just use a timestamp now, updated